src/library/scala/collection/immutable/TreeSet.scala

```diff
@@ -53,6 +107,8 @@ object TreeSet extends ImmutableSortedSetFactory[TreeSet] {
 final class TreeSet[A] private (tree: RB.Tree[A, Unit])(implicit val ordering: Ordering[A])
   extends SortedSet[A] with SortedSetLike[A, TreeSet[A]] with Serializable {
 
+  //for serialisation computability
```

Comment:
How do you mean?

Reply:
If instead we make `tree: RB.Tree[A, Unit]` above into `private val tree: RB.Tree[A, Unit]`, we end up renaming the its backing field to an expanded name (expanded because it is accessed from another class and `madeNotPrivate`). This changes the serialization fingerprint of the class.

2.13 collections use the serialization proxy pattern pervasively to avoid relying on default serialization.